frontend/test/metabase/scenarios/filters/relative-datetime.cy.spec.js

```diff
@@ -158,6 +158,113 @@ describe("scenarios > question > relative-datetime", () => {
         cy.findByText("days").should("exist");
       });
     });
+
+    it("should change the starting from units to match (metabase#22222)", () => {
+      openOrdersTable();
+
+      openCreatedAt("Past");
+      addStartingFrom();
+      popover().within(() => cy.findByText("days").click());
+      popover()
+        .last()
```

Comment:
Using `data-testid` for cases with multiple popovers would be more understandable here and below

Reply:
I couldn't add test IDs to the `Select` component cleanly so I separated the logic out into functions with more descriptive names